cni/pkg/ambient/net_linux.go

```diff
@@ -456,6 +456,19 @@ func (s *Server) CreateRulesOnNode(ztunnelVeth, ztunnelIP string, captureDNS boo
 			"--mark", constants.OutboundMark,
 			"-j", "ACCEPT",
 		),
+
```

Comment:
@bleggett Q - do we need a flag like workloadEntryEnabled to guard these cni changes?

Reply:
I don't think so, WEs are not an optional feature for ambient.

Reply:
Ok thanks!